### PR TITLE
python3Packages.coordinates: init at 0.4.0

### PR DESCRIPTION
**Pull request by mildlyincompetent**
###### Motivation for this change

Packaging coordinates, a python module.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mildlyincompetent** [comment]
@GrahamcOfBorg build python36Packages.coordinates python37Packages.coordinates python38Packages.coordinates

File: pkgs/development/python-modules/coordinates/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "coordinates";
+  version = "0.4.0";
+
```
Comment (bcdarwin) [inline pkgs/development/python-modules/coordinates/default.nix]: Please set `disabled = !isPy3k;`.

File: pkgs/development/python-modules/coordinates/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ynsrymapazhs03p6yxbcdhbq5ygdwdnkzab7jmpp6whhd0kf520";
+  };
+
+  # No tests in PyPi source distribution.
+  doCheck = false;
```
Comment (bcdarwin) [inline pkgs/development/python-modules/coordinates/default.nix]: I see you've requested these be bundled with the PyPi dist, but possibly consider switching to the GitHub release for now?

File: pkgs/development/python-modules/coordinates/default.nix (hunk 1 above)
Comment (bcdarwin) [reply pkgs/development/python-modules/coordinates/default.nix]: Sorry, also needs to be disabled for Python 3.5.

**Review by bcdarwin — CHANGES_REQUESTED** [review]
Thanks!  See a couple comments, mainly disabling for unsupported Pythons.  Python 3 version build on my NixOS machine.

**Comment by bcdarwin** [comment]
Also, please update the PR/commit to a buildable package name, here `python3Packages.coordinates: ...`.  Thanks!

File: pkgs/development/python-modules/coordinates/default.nix (hunk 1 above)
Comment (jonringer) [reply pkgs/development/python-modules/coordinates/default.nix]: `disabled = pythonOlder "3.5";`

**Review by jonringer — COMMENTED** [review]
Also, the commit message should be:
```
pythonPackages.coordinates: init at 0.4.0
```

**Comment by mildlyincompetent** [comment]
@GrahamcOfBorg build python3Packages.coordinates python36Packages.coordinates python37Packages.coordinates python38Packages.coordinates

**Comment by mildlyincompetent** [comment]
Thanks for the advice everyone, I've
- Switched to the github release, as that includes test
- Disabled for python 3.5 and below
- Fixed the commit message

Let me know if there's anything further.

File: pkgs/development/python-modules/coordinates/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    owner = "clbarnes";
+    repo = "coordinates";
+    rev = "v${version}";
+    sha256 = "1zha594rshjg3qjq9mrai2hfldya282ihasp2i3km7b2j4gjdw2b";
+  };
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest tests/
```
Comment (bcdarwin) [inline pkgs/development/python-modules/coordinates/default.nix]: should be (not ideal admittedly)

checkPhase = ''
  runHook preCheck
  pytest tests/
  runHook postCheck
'';

**Review by bcdarwin — APPROVED** [review]

File: pkgs/development/python-modules/coordinates/default.nix (hunk 3 above)
Comment (jonringer) [reply pkgs/development/python-modules/coordinates/default.nix]: in most cases, the explicit hooks aren't super necessary

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM
has tests :+1: 

```
[4 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/77335
2 package were built:
python37Packages.coordinates python38Packages.coordinates
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python37Packages.coordinates python38Packages.coordinates

**Pull request merged**